File: source/examples/MultiUser/AddUser/AddUser.java
App app = new App(new AppConfiguration.Builder("myapp-abcde").build());

// Log in as Joe
Credentials joeCredentials = Credentials.emailPassword("joe@example.com", "passw0rd");
app.loginAsync(joeCredentials, it -> {
  if (it.isSuccess()) {
    // The active user is now Joe
    User joe = it.get();
    assert joe == app.currentUser();
  } else {
    Log.e(TAG, it.getError().toString());
  }
});

// Log in as Emma
Credentials emmaCredentials = Credentials.emailPassword("emma@example.com", "pa55word");
app.loginAsync(emmaCredentials, it -> {
  if (it.isSuccess()) {
    // The active user is now Emma
    User emma = it.get();
    assert emma == app.currentUser();
  } else {
    Log.e(TAG, it.getError().toString());
  }
});

File: source/examples/MultiUser/AddUser/AddUser.kt
val app: App = App(AppConfiguration.Builder("<your app ID>").build())

// Log in as Joe
val joeCredentials: Credentials = Credentials.emailPassword("joe@example.com", "passw0rd")
app.loginAsync(joeCredentials) {
  if (it.isSuccess) {
    val joe: User = it.get()
    if (app.currentUser().email == joe.email) {
      Log.v(TAG,"Current user is joe")
    } else {
      Log.e(TAG, "Current user is not joe")
    }
  } else {
    Log.e(TAG, it.error));
  }
}

// Log in as Emma
val emmaCredentials: Credentials = Credentials.emailPassword("emma@example.com", "pa55word")
app.loginAsync(emmaCredentials) {
  if (it.isSuccess) {
    val emma: User = it.get()
    if (app.currentUser().email == emma.email) {
      Log.v(TAG,"Current user is emma")
    } else {
      Log.e(TAG, "Current user is not emma")
    }
  } else {
    Log.e(TAG, it.error);
  }
}
